src/fhirtypes/ElementDefinition.ts

```diff
@@ -820,14 +940,19 @@ export class ElementDefinition {
         valueCode: 'draft'
       };
     }
+
+    const connectedElements = this.findConnectedElements();
     if (mustSupport === true) {
       this.mustSupport = mustSupport;
+      connectedElements.forEach(ce => (ce.mustSupport = mustSupport || ce.mustSupport));
     }
     if (summary === true) {
       this.isSummary = summary;
+      connectedElements.forEach(ce => (ce.isSummary = summary));
     }
     if (modifier === true) {
       this.isModifier = modifier;
+      connectedElements.forEach(ce => (ce.isModifier = modifier || ce.isModifier));
     }
     if (newStatusExtension) {
```

Comment:
One more question I have, should the standards status flags be applied to connected elements in the same way as the old flags?

Reply:
That's... a good question. They're allowed to be different, but they'll usually be the same, so I'd say yes, they should get applied to connected elements.

Reply:
Update: got a negative response when consulting an expert, so no, they will not be propagated.

Reply:
Ok so no prop.